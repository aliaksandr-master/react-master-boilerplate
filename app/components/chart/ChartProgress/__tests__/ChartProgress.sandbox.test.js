/* eslint-env browser, jest */
/* eslint-disable react/jsx-no-bind */

import React from 'react';
import renderer from 'react-test-renderer';
import { wrapSandbox } from '../../../../vendor/react-sandbox/sandbox-staging';
import sandbox from '../ChartProgress.sandbox';
import Test from '../../../test/Test';



describe('rendering', () => {
  wrapSandbox(sandbox).forEach(({ name, template, props }) => {
    it(`should render sandbox "${name}"`, () => {
      const FieldAddKeywordsSandboxComponent = template();

      const component = renderer.create(
        <Test>
          <FieldAddKeywordsSandboxComponent {...props} />
        </Test>
      );

      expect(component.toJSON()).toMatchSnapshot();
    });
  });
});
