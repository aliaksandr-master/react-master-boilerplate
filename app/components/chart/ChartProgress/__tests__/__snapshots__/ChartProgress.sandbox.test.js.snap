// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering should render sandbox "default" 1`] = `
<div
  className="ChartProgress ChartProgress--themeDefault"
>
  <div
    className="ChartProgress__progress"
    style={
      Object {
        "background": "red",
        "height": 12,
        "width": "20%",
      }
    }
  >
    0.2
  </div>
</div>
`;

exports[`rendering should render sandbox "over" 1`] = `
<div
  className="ChartProgress ChartProgress--themeDefault"
>
  <div
    className="ChartProgress__progress"
    style={
      Object {
        "background": "red",
        "height": 12,
        "width": "100%",
      }
    }
  >
    1
  </div>
</div>
`;
